### PR TITLE
set working directory for node processes in supervisor

### PR DESCRIPTION
**Pull request by yihuang**
It's more convenient to config file streamer output relative to home directory.

### CODE REVIEW

File: pystarport/cluster.py [hunk 1]
```diff
@@ -267,7 +267,8 @@ def custom_edit_tm(doc):
         ini[section].update(
             dict(
                 COMMON_PROG_OPTIONS,
-                command=f"{self.cmd} start --home %(here)s/node{i}",
+                directory=f"%(here)s/node{i}",
+                command=f"{self.cmd} start --home .",
```
Comment (yihuang) [inline pystarport/cluster.py:271]:
```suggestion
                command="{self.cmd} start --home .",
```

File: pystarport/cluster.py [hunk 2]
```diff
@@ -1028,7 +1027,8 @@ def supervisord_ini(cmd, validators, chain_id, start_flags=""):
     for i, node in enumerate(validators):
         ini[f"program:{chain_id}-node{i}"] = dict(
             COMMON_PROG_OPTIONS,
-            command=f"{cmd} start --home %(here)s/node{i} {start_flags}",
+            directory=f"%(here)s/node{i}",
+            command=f"{cmd} start --home . {start_flags}",
```
Comment (yihuang) [inline pystarport/cluster.py:1031]:
```suggestion
            command="{cmd} start --home . {start_flags}",
```

File: pystarport/cluster.py [hunk 3]
```diff
@@ -1057,8 +1057,9 @@ def supervisord_ini_group(chain_ids):
     }
     cfg["program:relayer-demo"] = dict(
         COMMON_PROG_OPTIONS,
-        command=("hermes --config %(here)s/relayer.toml start"),
-        stdout_logfile="%(here)s/relayer-demo.log",
+        directory="%(here)s",
+        command=("hermes --config relayer.toml start"),
```
Comment (yihuang) [inline pystarport/cluster.py:1061]:
```suggestion
        command="hermes --config relayer.toml start",
```

**Review by mmsqe — APPROVED** [review]

File: pystarport/cluster.py [hunk 4]
```diff
@@ -1028,7 +1027,8 @@ def supervisord_ini(cmd, validators, chain_id, start_flags=""):
     for i, node in enumerate(validators):
         ini[f"program:{chain_id}-node{i}"] = dict(
             COMMON_PROG_OPTIONS,
-            command=f"{cmd} start --home %(here)s/node{i} {start_flags}",
+            directory=f"%(here)s/node{i}",
+            command="{cmd} start --home . {start_flags}",
```
Comment (yihuang) [inline pystarport/cluster.py:1031]:
```suggestion
            command=f"{cmd} start --home . {start_flags}",
```

File: pystarport/cluster.py [hunk 5]
```diff
@@ -267,7 +267,8 @@ def custom_edit_tm(doc):
         ini[section].update(
             dict(
                 COMMON_PROG_OPTIONS,
-                command=f"{self.cmd} start --home %(here)s/node{i}",
+                directory=f"%(here)s/node{i}",
+                command="{self.cmd} start --home .",
```
Comment (yihuang) [inline pystarport/cluster.py:271]:
```suggestion
                command=f"{self.cmd} start --home .",
```

File: pystarport/cluster.py [hunk 6]
```diff
@@ -1057,8 +1057,9 @@ def supervisord_ini_group(chain_ids):
     }
     cfg["program:relayer-demo"] = dict(
         COMMON_PROG_OPTIONS,
-        command=("hermes --config %(here)s/relayer.toml start"),
-        stdout_logfile="%(here)s/relayer-demo.log",
+        directory="%(here)s",
+        command="hermes --config relayer.toml start",
+        stdout_logfile="relayer-demo.log",
```
Comment (mmsqe) [inline pystarport/cluster.py:1062]: miss directory, but why need change replayed one?

Comment (yihuang) [reply pystarport/cluster.py:1062]: I haven't tested, we can revert this

Comment (yihuang) [reply pystarport/cluster.py:1062]: The changes are being tested here: https://github.com/crypto-org-chain/cronos/pull/722
If the integration tests success, I guess this PR is safe to merge.

**Pull request merged**